webapp/src/components/AssetPage/SaleRentActionBox/SaleRentActionBox.tsx

```diff
@@ -80,169 +83,201 @@ const SaleRentActionBox = ({
 
   const handleCloseAuthorizationModal = () => setShowAuthorizationModal(false)
 
+  const rentalEndTime =
+    rental && rental.startedAt && rental.rentedDays
```

Comment:
```suggestion
   isBeingRented ? add(rental.startedAt, { days: rental.rentedDays }).getTime() : 0
```
I'd use `isBeingRented` here, looks a bit cleaner, wdyt?

Reply:
I've split it because I had to use `!` on `rent` and on `startedAt` because the compiler doesn't know the value of the variable, but I've changed it to use `isBeingRented` and used `!`, let me know if it seems better.

Reply:
looks great!